projects/quml-library/src/lib/section-player/section-player.component.ts

```diff
@@ -216,7 +216,7 @@ export class SectionPlayerComponent implements OnChanges, AfterViewInit {
     this.showTimer = this.sectionConfig.metadata?.showTimer;
     
     //server-level-validation
-    this.questionSetEvaluable = this.viewerService.serverValidationCheck(this.sectionConfig.metadata?.eval);
+    this.questionSetEvaluable = this.viewerService.serverValidationCheck(this.sectionConfig.metadata?.evalMode);
```

Comment:
@pavankumar0408 Instead of calling `this.viewerService.serverValidationCheck` method can you set the value of `questionSetEvaluable` in viewerService one time and use it like `this.viewerService.questionSetEvaluable` in all the components

Reply:
done